test/python/fe_api/gemm/test_gemm_srelu_dsrelu_jax.py

```
@@ -0,0 +1,139 @@
+
+jax = pytest.importorskip("jax")
+ml_dtypes = pytest.importorskip("ml_dtypes")
+torch = pytest.importorskip("torch")
+import jax.numpy as jnp
+
+from fe_api.gemm.test_gemm_amax_jax import device_sync, skip_unless_sm100
+
+from cudnn.api_base import ceil_div
+
+
+def make_inputs(m, n, k, l, sf_vec_size, rng):
```

Ruff (0.16.1)

[error] Ambiguous variable name: `l`

(E741)